src/cpp/redisserver.cpp

```diff
@@ -52,8 +54,19 @@ RedisServer::RedisServer()
 
     _command_attempts = (_command_timeout * 1000) /
                          _command_interval + 1;
+
+    _tp = new ThreadPool(_thread_count);
```

Comment:
Following up from a previous comment, is there any overhead associated with creating a ThreadPool? I worry a little bit that client initialization is beginning to take a long time. While the cost is amortized over the length of the run, it's still something we need to keep an eye on.

Reply:
Do we have any performance tests measuring the initialization time, so that we can keep track of this over time?

Reply:
If not, maybe we can introduce a `#TIME_THREADPOOL_INIT` segment to enable that in the future?

Reply:
The overhead wasn't noticeable when I watched the tests running on my Mac. I would guess it's on the order of a millisecond or two right now, not nearly enough to worry about

Reply:
The overhead wasn't noticeable when I watched the tests running on my Mac. I would guess it's on the order of a millisecond or two right now, not nearly enough to worry about